Day1/P1.py

```diff
@@ -0,0 +1,154 @@
+#string find method
+txt = 'Hello my name is Tram'
+print(txt.find('a'))
+print(txt.find('my'))
+
+#find the position of the first occurrence of a character in a string
+def findchar(txt,char,start=None,end=None):
+    myindex = -1
+    if start is None:
+        start = 0
+    if end is None:
+        end = len(txt)
+    for i in range(start,end):
+        if txt[i:i+len(char)] == char:
+            myindex = i
+            break
+    return myindex
+
+print(findchar('hello my name is tram','a'))
+print(findchar('hello my name is tram','my'))
+print(findchar('hello my name is tram','z'))
+print(findchar('hello my name is tram','lo'))
+print(findchar('hello my name is tram','a',12))
+print(findchar('hello my name is tram','m',8,12))
+print('-'*20)
+
+#find the positions of all occurrences of a character in a string
+def findchars(txt,char,start=None,end=None):
+    myindex = []
+    if start is None:
+        start = 0
+    if end is None:
+        end = len(txt)
+    for i in range(start,end):
+        if txt[i:i+len(char)] == char:
+            myindex.append(i)
+    return myindex
+
+print(findchars('hello my name is tram','a'))
+print(findchars('hello my name is tram','my'))
+print(findchars('hello my name is tram','z'))
+print(findchars('hello my name is tram','lo'))
+print(findchars('hello my name is tram','a',12))
+print(findchars('hello my name is tram','m',8,12))
+print(findchars('ohmymymyabcdefmy','my'))
+print('-'*20)
+
+#Convert string into lower case
+txt = 'Hello, And Welcome To My World!'
+print(txt.lower())
+
+def lowerstr(txt):
+    uppertolower = {'A':'a','B':'b','C':'c','D':'d','E':'e','F':'f','G':'g','H':'h','I':'i','J':'j','K':'k','L':'l','M':'m','N':'n','O':'o','P':'p','Q':'q','R':'r','S':'s','T':'t','U':'u','V':'v','W':'w','X':'x','Y':'y','Z':'z'}
+    for i in range(len(txt)):
+        if txt[i] in uppertolower.keys():
```

Comment:
You are checking whether an element is in a dictionary or not. This syntax is not wrong, but it is too complicated. There is a more standard way to do element checking with dictionary.